### PR TITLE
Export releases to TOML #2

### PR DESCRIPTION
**Pull request by mosteo**
Second half of good changes

### CODE REVIEW

File: src/alire/alire-properties-labeled.ads [hunk 1]
```diff
@@ -88,6 +90,16 @@ private
...
 
+   overriding function Key (L : Label) return String is
+     (case L.Name is
+         when Author       => "authors",
+         when Comment      => "comment",
+         when Executable   => "executables",
+         when Maintainer   => "maintainers",
+         when Path         => "paths",
+         when Project_File => "project-files",
+         when Website      => "website");
```
Comment (pmderodat) [inline src/alire/alire-properties-labeled.ads]: Looks like this should directly reference `Alire.TOML_Keys`.

File: src/alire/alire-properties-licenses.ads [hunk 2]
```diff
@@ -1,11 +1,17 @@
 with Alire.Licensing;
+with Alire.TOML_Keys;
 
 package Alire.Properties.Licenses is
 
    function Image (L : Licensing.Licenses) return String;
 
+   pragma Warnings (Off);
+   function Key (L : Licensing.Licenses) return String is (TOML_Keys.License);
+   pragma Warnings (On); -- Unreferenced => L
```
Comment (pmderodat) [inline src/alire/alire-properties-licenses.ads]: Protip: rename `L` to `Dummy` (or `Dummy_L`), and the warning will go away.

File: src/alire/alire-releases.adb [hunk 3]
```diff
@@ -459,42 +459,44 @@ package body Alire.Releases is
...
+--                                        when Multiple => Values.To_TOML);
+--                 begin
+--                    General.Set (TOML_Keys.Labels (Label),
+--                                 TOML_Values);
+--                 end;
+--              end if;
+--           end;
+--        end loop;
+--
+--        Root.Set ("general",       General);
```
Comment (pmderodat) [inline src/alire/alire-releases.adb]: Have you left this by accident? Commented code looks messy, and no explanation about why it’s there will create doubt in the future when we’ll wonder if we have to leave it here or remove it ^^

File: src/alire/alire-dependencies.adb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+with Alire.TOML_Adapters;
+
+package body Alire.Dependencies is
+
+   overriding function To_TOML (Dep : Dependency) return TOML.TOML_Value is
+      use Semantic_Versioning;
+      use TOML_Adapters;
+   begin
+      if Dep.Versions = Any then
+         return +"*";
```
Comment (pmderodat) [inline src/alire/alire-dependencies.adb]: The string should be `any` (that’s what `Alire.TOML_Index` expects)

File: src/alire/alire-properties-labeled.ads (hunk 1 above)
Comment (mosteo) [reply src/alire/alire-properties-labeled.ads]: Indeed!

File: src/alire/alire-properties-licenses.ads (hunk 2 above)
Comment (mosteo) [reply src/alire/alire-properties-licenses.ads]: Is this some gnat magic? :D

File: src/alire/alire-releases.adb (hunk 3 above)
Comment (mosteo) [reply src/alire/alire-releases.adb]: Right you are, to be deleted

File: src/alire/alire-properties-licenses.ads (hunk 2 above)
Comment (pmderodat) [reply src/alire/alire-properties-licenses.ads]: Yes indeed. ;-) See https://docs.adacore.com/gnat_rm-docs/html/gnat_rm/gnat_rm/implementation_defined_pragmas.html#pragma-unmodified

**Pull request merged**